server/db.sql

```diff
@@ -0,0 +1,101 @@
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 13.3
+-- Dumped by pg_dump version 14.2
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+SET default_tablespace = '';
+
+SET default_table_access_method = heap;
+
+--
+-- Name: users; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.users (
+    id SERIAL PRIMARY KEY,
+    email VARCHAR(255) UNIQUE NOT NULL,
+    created_at timestamp NOT NULL DEFAULT NOW(),
+    updated_on timestamp default CURRENT_TIMESTAMP not null
+);
+
+--
+-- Name: spices; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.spices (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+    brand VARCHAR(255),
+    full_weight VARCHAR(255),
+    current_weight VARCHAR(255),
+    expiration_date DATE,
+    last_purchased DATE,
+    notes VARCHAR(255),
+    inactive BOOLEAN DEFAULT FALSE, 
+    user_id INTEGER REFERENCES users(id),
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    updated_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+--
+-- Name: shopping_list; Type: TABLE; Schema: public; Owner: -
+--
+CREATE TABLE public.shopping_list(
+    id SERIAL PRIMARY KEY,
+    spice_id INTEGER REFERENCES spices(id),
+    user_id INTEGER REFERENCES users(id) NOT NULL,
+    shopping_date DATE,
+    comments TEXT,
+    created_at timestamp NOT NULL DEFAULT NOW(),
+    updated_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
+);
+
+
+--
+-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: -
+--
+INSERT INTO public.users (email)
```

Comment:
Are these inserts in place for development, or are you intending to have a fixed set of users, spices and shopping lists in production as well?